.gitignore

```diff
@@ -30,3 +30,6 @@ var/
 # protostar
 !contracts/build/
 contracts/build/*[!_abi].json
+
+# cli
+cli-config.ini
```

Comment:
why not include the config with different gateway urls etc as part of the package?

Reply:
it will have secrets in it